config/default.json

```diff
@@ -9,6 +9,15 @@
     "logger": {
       "level": "info",
       "prettyPrint": false
+    },
+    "tracing": {
+      "enabled": "false",
+      "url": "http://localhost:4318/v1/traces"
+    },
+    "metrics": {
+      "enabled": false,
+      "url": "http://localhost:4318/v1/metrics",
+      "interval": 5
```

Comment:
interval is being used? where?

Reply:
interval value seems to be too low because its presenting MS. consider change to higher number of milliseconds.